From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 27 Sep 2020 12:05:04 +0100
Subject: ROOT=/.

---
 go.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/go.sh b/go.sh
index c567b67..683e798 100755
--- a/go.sh
+++ b/go.sh
@@ -24,7 +24,7 @@ set -e
 #    files (e.g. signed public releases should match signed intel releases).
 #    "publish" rule. 
 
-ROOT=
+ROOT=/.
 INTEL_PATH=lib/firmware/intel
 VERSION=$(git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3| cut -d"-" -f 2)
 
