From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 27 Sep 2020 12:18:13 +0100
Subject: explicit version

---
 go.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/go.sh b/go.sh
index 683e798..a9bb612 100755
--- a/go.sh
+++ b/go.sh
@@ -26,7 +26,9 @@ set -e
 
 ROOT=/.
 INTEL_PATH=lib/firmware/intel
-VERSION=$(git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3| cut -d"-" -f 2)
+VERSION=v1.5.1
+# consider tagging and using some variant of
+#VERSION=$(git describe HEAD ...)
 
 echo "Installing Intel firmware and topology $VERSION to $INTEL_PATH"
 
